docker/README.md

```diff
@@ -0,0 +1,46 @@
+## docker installation
+
+1. build the tradenote image with your custom parameters
+```
+docker build . -t tradenote --build-arg PARSE_INIT=APPLICATION_ID --build-arg PARSE_URL=http://pi:1337/parse
+```
+
+2. run mongodb, parse server tradenote via docker compose file
+```
+docker-compose up -d
+```
+
+3. modify PARSE_INIT, PARSE_MASTER_KEY, PARSEHOST in addclass.sh to match your environment, run addclass.sh to add classes
```

Comment:
You also created a modifyuser.sh. Is it working ? If so, would be good to add it here. If not, maybe remove the modifyuser.sh file so that it's less confusing ?

Reply:
the "modifyuser.sh" only works for database ( I mean it can add data to mongodb, but not show in the dashboard, I cannt login to tradenote ), so users need step 6 to add columns and data to _User class

why I still pr this file, I am thinking maybe you or someone can find out a solution for this, so I can merge 2 scripts :)
if this is confusing, I will remove this script